src/resources/js/components/enso/select/VueSelect.vue

```diff
@@ -438,7 +442,9 @@ export default {
         },
         optionLabel(option, label) {
             return label.split('.')
-                .reduce((result, property) => result[property], option);
+                .reduce((result, property) =>
+                    (this.translateFields)? this.i18n(result[property]) : result[property]
+                    , option);
```

Comment:
this can potentially break the logic.

instead do
```vue
optionLabel(option, label) {
        optionLabel = label.split('.')
                .reduce((result, property) => result[property], option);
                .reduce((result, property) => result[property], option);
      
     return this.translated
            ? this.i18n(optionLabel)
             : optionLabel;
```